app/src/main/java/com/tvtracker/ui/main/BrowseViewModel.kt

```diff
@@ -18,11 +18,11 @@ import kotlinx.coroutines.launch
 
 class BrowseViewModel(var mediaService: IMediaService = MediaService()): ViewModel() {
 
-    private lateinit var firestore: FirebaseFirestore
```

Comment:
LateInit was unnecessary for the it was definitely initialized in the constructors